libc/src/math/faddf128.h

```diff
@@ -0,0 +1,22 @@
+//===-- Implementation of faddf128 function -------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/__support/macros/config.h"
+#include "src/__support/macros/properties/types.h"
+
+
+#ifndef LLVM_LIBC_SRC_MATH_FADDF128_H
+#define LLVM_LIBC_SRC_MATH_FADDF128_H
```

Comment:
See https://github.com/llvm/llvm-project/pull/99694#discussion_r1685059983. You should probably just recreate the libc/src/math/fadd*.h headers with https://github.com/overmighty/llvm-dev-utils/blob/master/llvm_libc_add_math_function.py.